### PR TITLE
mobile: new onboarding

### PR DESCRIPTION
**Pull request by dcposch**
image

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/onboarding/OnboardingChooseNameScreen.tsx [hunk 1]
```diff
@@ -79,12 +79,30 @@ export function OnboardingPickNameScreen({ route }: Props) {
   );
 }
 
-function IconRow(props: { icon: OctName; color?: string; title: string }) {
+function CoverImage() {
+  return (
+    <View style={{ display: "flex", alignItems: "center" }}>
+      <Image source={ImgChooseName} style={{ width: 188, height: 156 }} />
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/onboarding/OnboardingChooseNameScreen.tsx:85]: we have a similar `CoverGraphic` as well, might be able to cleanly merge?

File: apps/daimo-mobile/src/view/sheet/CreateBackupSheet.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  return (
+    <AddKeySlotButton
+      buttonTitle="Backup with Passkey"
+      account={account}
+      slot={passkeySlot}
+    />
+  );
+}
+
+function AddSecurityKeyButton() {
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/sheet/CreateBackupSheet.tsx:52]: these two buttons can be merged easily? param SlotType

File: packages/daimo-common/src/keySlot.ts [hunk 3]
```diff
@@ -1,35 +1,48 @@
...
+// slots 0 - 63 are for mobile devices
+// slots 64 - 127 are for computer devices
+// slots 128 - 160 are for passkey backups
+// slots 160 - 192 are for security key backups
+// slots 192 - 256 are for seed phrase backups
 const slotTypeToFirstSlot = {
   [SlotType.Phone]: 0,
   [SlotType.Computer]: 0x40,
   [SlotType.PasskeyBackup]: 0x80,
+  [SlotType.SecurityKeyBackup]: 0xa0, // TODO: check max used slot in prod < 0xa0
```
Comment (nalinbhardwaj) [inline packages/daimo-common/src/keySlot.ts:21]: safe to assume this never happened, can drop comment

File: packages/daimo-userop/src/daimoOpSender.ts [hunk 4]
```diff
@@ -59,15 +59,15 @@ export class DaimoOpSender {
...
-    const { tokenAddress, tokenDecimals } = opConfig;
-    console.log(
-      `[OP] init: ${JSON.stringify({
-        accountAddress,
-        tokenAddress,
-        tokenDecimals,
-        notesAddressV1: opConfig.notesAddressV1,
-      })})}`
-    );
+    // const { tokenAddress, tokenDecimals } = opConfig;
```
Comment (nalinbhardwaj) [inline packages/daimo-userop/src/daimoOpSender.ts:62]: uncomment?

File: apps/daimo-mobile/src/view/screen/onboarding/OnboardingChooseNameScreen.tsx (hunk 1 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/screen/onboarding/OnboardingChooseNameScreen.tsx:85]: fixed

File: apps/daimo-mobile/src/view/sheet/CreateBackupSheet.tsx (hunk 2 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/sheet/CreateBackupSheet.tsx:52]: done

**Comment by dcposch** [comment]
## Screenshots

Screenshot 2024-04-16 at 01 00 23
Screenshot 2024-04-16 at 01 01 29
Screenshot 2024-04-16 at 01 02 24
Screenshot 2024-04-16 at 01 07 36
Screenshot 2024-04-16 at 01 07 42
Screenshot 2024-04-16 at 01 07 45
Screenshot 2024-04-16 at 01 07 51
Screenshot 2024-04-16 at 01 07 51

**Comment by dcposch** [comment]
## Android screenshots

Screenshot 2024-04-16 at 02 09 52
Screenshot 2024-04-16 at 02 09 55
Screenshot 2024-04-16 at 02 10 04
Screenshot 2024-04-16 at 02 10 13
Screenshot 2024-04-16 at 02 10 40
Screenshot 2024-04-16 at 02 10 47
Screenshot 2024-04-16 at 02 11 34

**Comment by dcposch** [comment]
Tested, works on Android. Shipping

**Pull request merged**